rhv_checks.py

```diff
@@ -235,6 +236,63 @@ def check_storage_domain_attached_status(system, **kwargs):
         print("Ok: all Storage Domain(s) are Attached to Data Center(s): {}".format(okay))
         sys.exit(0)
 
+
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the hosts."""
+    okay, warning, critical, unknown, all = [], [], [], [], []
+    hosts = system.api.system_service().hosts_service().list()
+
+    for host in hosts:
+        result = ssh_command(host.name, password, command)
+        host_data_list = []
+        host_data = {}
```

Comment:
Can you add some comments around what `host_data_list` and `host_data`? What exactly do you want it to look like?

Reply:
Update logic.